mosquitto/config/mosquitto.conf

```diff
@@ -0,0 +1,914 @@
+# Config file for mosquitto
```

Comment:
If you want 'hab pkg config` to display these comments for helper options you need to move them into `default.toml`

Reply:
This is the default configuration file for mosquitto. The comments are from them. Don't know if it's useful to copy them. what do you think?

Reply:
I'm of the opinion we should match the upstream default configuration as close as possible, comments and all, but I'd love to hear the rest of the maintainers thoughts on it.

Reply:
I am in agreement here with @sdmacfarlane. The closer we can get to typical distro out of the box experience the better. It makes the config files uglier to write but as a user with limited hab internals knowledge it might be better to have those in there.